packages/app/pages/api/legacy-endpoints/getModel.ts

```diff
@@ -0,0 +1,22 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import getModelHandler from '../models/[modelName]/'
+import { z } from 'zod'
+import { withApiErrorHandler } from 'lib/with-api-error-handler'
+
+const schema = z
```

Comment:
Annoyingly, the query param is "name" instead of "model" for only this one endpoint. Do you know of a way to combine them, just changing the object property to "name"?

Reply:
Ah, okay—I just didn't notice the different param name! This works totally fine, of course. But I guess if you wanted to reuse the logic, I'd probably do something like this, since Zod doesn't throw for extra properties because we're not using `strict()`:

```js
// this one endpoint uses "name" instead of "model"
req.query.model = req.query.name
// replaces query params with params mapped to RESTful names (e.g. "model" -> "modelName", etc.)
req.query = modelSchema.parse(req.query)
```

Reply:
That's much easier than managing another schema...d'oh

Made the change